app/views/admin/reports/index.html.haml

```diff
@@ -0,0 +1,41 @@
+%body
+  - content_for(:title) do
+    = 'Reports'
+  %header.main-content__header{ role: 'banner' }
+    %h1.main-content__page-title
+      = content_for(:title)
+    %div
+      = link_to t("administrate.actions.back"), :back, class: "button"
+  %section.main-content__body
+    %h1 Total Children Served: #{Need.total_children_served}
+    %hr
+    %h2 Office
+    %ul
+      %li= link_to('Total Volunteer Minutes', admin_reports_total_volunteer_minutes_by_office_path)
+      %li= link_to('Total Children Served', admin_reports_total_children_served_by_office_path)
+    %hr
+    %h2 State
+    %div= link_to('Total Volunteer Minutes', admin_reports_total_volunteer_minutes_by_state_path)
```

Comment:
I don't have much context here since I'm not actually running this code, but why does the `Office` section use a `ul` and the `State` section doesn't?